src/parseOption.ts

```diff
@@ -106,7 +107,7 @@ export function getValue(itemData: object, field: string) {
     console.warn('[Tooltip] Cannot find field ' + field + ' in data.');
     return undefined;
   } else {
-    return value;
+    return value as string | number | Date;
```

Comment:
Because otherwise when we return to the caller, we can assume that it is already either a string, number or date. What the above `accessors` function on line 98 does is to repeatedly access the key until we get down to the innermost value. But there's no way I could think of to declare as a type, so I have to use type assertion.